### ISSUE TITLE
UI Overlap

### ISSUE THREAD

**Issue by junebug12851** [junebug12851/pokered-save-editor#6]
On many sizes the UI controls will overlap one another, sometimes badly. This can currently be corrected by resizing larger or smaller and this will be fixed and addresses soon with a UI revamp.

EDIT:
To clarify if your having this issue, a temporary workaround is by changing the window size the controls will snap to a new size and often will eliminate the overlapping.

**Comment by junebug12851** [comment]
The problem is I really need someone to overhaul the UI system properly. I've made 2 or 3 overhauls but they've all had various issues with them that can be frustrating to work with. I've tried 2 or 3 kinds of UI frameworks as well and am mostly at a loss on this. Every overhaul causes me to work overtime on the project and it often kind of flops at the end.

I've had so much issues with the UI I've actually considered several times to move off HTML5 entirely but I really could use some help on this.
